packages/theme/src/theme/_forms.checkbox.scss

```diff
@@ -47,48 +31,64 @@ input[type='checkbox'],
 	&:hover {
 		&:before,
 		+ *:before {
-			border-color: $silver-chalice;
+			border-color: $tt-checkbox-hover-border-color;
+		}
+	}
+
+	&:focus {
+		&:before,
+		+ *:before {
+			// border-color: $tt-checkbox-focus-border-color;
+			@include checkbox-outline;
 		}
 	}
 
+	// Indeterminate Checkboxes style
+	&[data-checked='1']:before,
+	&[data-checked='1'] + *:before {
+		background-color: $tt-checkbox-checked-background-color;
+		border-color: $tt-checkbox-checked-border-color;
```

Comment:
is it normal that the border is different in indeterminate and checked case ? 
![image](https://user-images.githubusercontent.com/14272767/64427422-7e675200-d0b1-11e9-8ef2-1189caf626be.png)

if yes, seems like the same variable is used for both whereas guidelines says it should be shade in one case and tint in another

Reply:
Hello @jlame — there is a mistake on the frontify page we should only use "Tint" instead of "darker" values. I'll update the guidelines ASAP. Thank you :)